spec/features/admin_users_spec.rb

```diff
@@ -0,0 +1,107 @@
+require 'capybara_helper'
+
+describe Admin::UsersController do
```

Comment:
I think it was in Aarons railsconf keynote, but I did not find anything about it in the rails 5 changelog. Maybe his work did not make it into rails yet.